eradiate/tests/system/test_onedim_symmetry.py

```diff
@@ -83,7 +83,12 @@ def test_symmetry_zenith(mode_mono_double, surface, atmosphere):
         }[surface],
         atmosphere={
             "none": None,
-            "homogeneous": {"type": "homogeneous", "sigma_s": 1.0e-2},
+            "homogeneous": {
+                "type": "homogeneous",
+                "sigma_s": 1.0e-2,
+                "top": 100,
+                "top_units": "km",
+            },
```

Comment:
In order to characterise more reliably the test scene, I'd suggest:
```suggestion
            "homogeneous": {
                "type": "homogeneous",
                "sigma_s": 1.0e-2 * ureg.km ** -1,
                "top": 1.0e2 * ureg.km,
            },
```
This way, your test slab always has an optical thickness equal to 1, no matter what strange thing default units could do in the future.